src/eventSubscription/commands/mock/createMockEventGenerator.ts

```diff
@@ -58,6 +77,11 @@ export async function createMockEventGenerator(node?: IAzureNode<EventSubscripti
     eventSchema.properties.eventType.pattern = `${eventType.replace('.', '\\.')}\\.${eventSubTypePattern}`;
     eventSchema.properties.subject.pattern = subjectPattern;
 
+    const definitionsPath: string = path.join(templatesPath, 'definitions', `${fileName}.json`);
+    if (await fse.pathExists(definitionsPath)) {
+        eventSchema.definitions = <{}>await fse.readJson(definitionsPath);
```

Comment:
Question: How is <{}> different from the <any> cast?

Reply:
I'm not sure what you're asking here. readJson returns type of 'any' and I want this to be type of '{}'